1-mandatory/1-alarmclock/alarmclock.js

```diff
@@ -1,8 +1,48 @@
-function setAlarm() {}
+function setAlarm() {
+  
+    let timerId;
+    let flashingId = "";
+    
+    const inputTime = document.getElementById("alarmSet").value;
+    let remainTime = document.getElementById("timeRemaining");
+    remainTime.innerText = "Time Remaining: " + formatTime(inputTime);
+  
+    let alarmTime = parseInt(inputTime);
+    timerId = setInterval(() => {
+      alarmTime -= 1;
+      remainTime.innerText = "Time Remaining: " + formatTime(alarmTime);
+      if (alarmTime == 0) {
+        clearInterval(timerId);
+        playAlarm();
+        flashBgr();
+      }
+      function flashBgr() {
+        var x = Math.floor(Math.random() * 256);
+        var y = Math.floor(Math.random() * 256);
+        var z = Math.floor(Math.random() * 256);
+        var bgColor = "rgb(" + x + "," + y + "," + z + ")";
+     console.log(bgColor);
```

Comment:
Since the values of colours are not changed later is there a different way we can declare this?